[2024-12-05T22:55:27.638+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T22:55:27.650+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: brand_category_mapping.get_next_batch manual__2024-12-05T22:55:24.537689+00:00 [queued]>
[2024-12-05T22:55:27.656+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: brand_category_mapping.get_next_batch manual__2024-12-05T22:55:24.537689+00:00 [queued]>
[2024-12-05T22:55:27.656+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 4
[2024-12-05T22:55:27.673+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_next_batch> on 2024-12-05 22:55:24.537689+00:00
[2024-12-05T22:55:27.680+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=9000) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T22:55:27.682+0000] {standard_task_runner.py:72} INFO - Started process 9001 to run task
[2024-12-05T22:55:27.682+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'brand_category_mapping', 'get_next_batch', 'manual__2024-12-05T22:55:24.537689+00:00', '--job-id', '1967', '--raw', '--subdir', 'DAGS_FOLDER/llm_mapping.py', '--cfg-path', '/tmp/tmpkxijrhpa']
[2024-12-05T22:55:27.684+0000] {standard_task_runner.py:105} INFO - Job 1967: Subtask get_next_batch
[2024-12-05T22:55:27.727+0000] {task_command.py:467} INFO - Running <TaskInstance: brand_category_mapping.get_next_batch manual__2024-12-05T22:55:24.537689+00:00 [running]> on host c7780884450d
[2024-12-05T22:55:27.797+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='brand_category_mapping' AIRFLOW_CTX_TASK_ID='get_next_batch' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T22:55:24.537689+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T22:55:24.537689+00:00'
[2024-12-05T22:55:27.798+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T22:55:27.824+0000] {base.py:84} INFO - Retrieving connection 'database_cloud'
[2024-12-05T22:55:28.728+0000] {llm_mapping.py:60} INFO - Все бренды обработаны. Завершение работы DAG.
[2024-12-05T22:55:28.729+0000] {python.py:240} INFO - Done. Returned value was: {'batch': [], 'next_index': 5950}
[2024-12-05T22:55:28.748+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T22:55:28.749+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=brand_category_mapping, task_id=get_next_batch, run_id=manual__2024-12-05T22:55:24.537689+00:00, execution_date=20241205T225524, start_date=20241205T225527, end_date=20241205T225528
[2024-12-05T22:55:28.780+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T22:55:28.799+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T22:55:28.802+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
